src/macros-by-example.md

```diff
@@ -195,6 +195,220 @@ compiler knows how to expand them properly:
     not have the same number. This requirement applies to every layer of nested
     repetitions.
 
+## Meta variable expressions
```

Comment:
"Meta variable" should be "metavariable" for consistency with everything else. Ditto for all other appearances of the phrase.